### ISSUE TITLE
Class 'Crypt_CHAP_MSv2' not found error when using composer autoload.php

### ISSUE THREAD

**Issue by malle-pietje** [dapphp/radius#4]
I have checked and the `Pear_CHAP.php` file is available.
Any idea why this is happening?

Here's the code I'm using. The error occurs when using MSCHAPv2, PAP has been tested and works fine:
```php
        $radius = new \Dapphp\Radius\Radius();
        $radius->setServer($radius_server['server'])
               ->setSecret($radius_server['shared_secret'])
               ->setNasIpAddress($radius_server['nas_ip_address']);

        /**
         * Send auth request, select method based on protocol value
         * can be PAP, CHAP, MSCHAPv1 or MSCHAPv2
         */
        switch ($radius_server['protocol']) {
            case 'PAP':
                $response = $radius->accessRequest($_REQUEST['radius_username'], $_REQUEST['radius_password']);
                break;
            case 'CHAP':
                $radius->setChapPassword($_REQUEST['radius_password']);
                $response = $radius->accessRequest($_REQUEST['radius_username']);
                break;
            case 'MSCHAPv1':
                $radius->setMSChapPassword($_REQUEST['radius_password']);
                $response = $radius->accessRequest($_REQUEST['radius_username']);
                break;
            case 'MSCHAPv2':
                $response = $radius->accessRequestEapMsChapV2($_REQUEST['radius_username'], $_REQUEST['radius_password']);
                break;
            default:
                $response = $radius->accessRequest($_REQUEST['radius_username'], $_REQUEST['radius_password']);
                break;
        }
```

**Comment by malle-pietje** [comment]
I guess I should add; the `Class 'Crypt_CHAP_MSv2' not found` error only occurs when submitting valid credentials.

**Comment by malle-pietje** [comment]
I just tested the above using the provided `autoload.php` and that works fine.

Here's a copy of my composer.json file:
```javascript
{
    "require": {
        "art-of-wifi/unifi-api-client": "^1.1",
        "dapphp/radius": "2.5.2",
        "nesbot/carbon": "~1.21"
    }
}
```

**Comment by dapphp** [comment]
Hi @malle-pietje,

Thank you very much for spotting this and reporting it.

I've released 2.5.4 which fixes the issue.  Rather than statically autoload the file for all requests, I stubbed out placeholder files that require the appropriate Pear_CHAP.php file so PSR-0 autoloading can be used.  I also switched the test's bootstrap.php to use the composer autoloader instead of the old standalone one.

Let me know if you run into any issues with the new release.

Thanks!
